### PR TITLE
Add missing change log for 2.15 MDS PRs

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description

Adding missing change log for the following PRs:

https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6395
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385

## Changelog

- skip
### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: changelogs/fragments/6919.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+feat:
+- Modify the adding sample data part for timeline ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```
Comment (AMoo-Miki) [inline changelogs/fragments/6919.yml:2]: The entry should describe the outcome not what was done.

File: changelogs/fragments/6395.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Do not support import data source object to Local cluster when not enable data source ([#6395](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6395))
```
Comment (AMoo-Miki) [inline changelogs/fragments/6395.yml:2]: I am not sure about the context but is this maybe
```suggestion
- [MDS] Prevent importing of data source object when MDS is not enabled ([#6395](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6395))
```

File: changelogs/fragments/6385.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Support for Timeline ([#6385](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385))
```
Comment (AMoo-Miki) [inline changelogs/fragments/6385.yml:2]: While this is nice and short, it is not clear enough what impact users should expect.

File: changelogs/fragments/6919.yml (hunk 1 above)
Comment (zhyuanqi) [reply changelogs/fragments/6919.yml:2]: Thanks Miki. How about "Enable the addition of sample data for timeline visualizations with MDS enabled. "

File: changelogs/fragments/6919.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+feat:
+- Enable the addition of sample data for timeline visualizations with MDS enabled.  ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```
Comment (zhyuanqi) [inline changelogs/fragments/6919.yml]:
```suggestion
- [MDS] Enable the addition of sample data for timeline visualizations with MDS enabled.  ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```

File: changelogs/fragments/6385.yml (hunk 3 above)
Comment (zhongnansu) [reply changelogs/fragments/6385.yml]: updated

**Review by zhyuanqi — DISMISSED** [review]

File: changelogs/fragments/6385.yml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Enable user to query from data source in Timeline ([#6385](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385))
```
Comment (AMoo-Miki) [inline changelogs/fragments/6385.yml:2]:
```suggestion
- [MDS] Allow querying from data sources in Timeline visualizations ([#6385](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385))
```

File: changelogs/fragments/6919.yml [hunk 6]
```diff
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Enable the addition of sample data for timeline visualizations with MDS enabled ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```
Comment (AMoo-Miki) [inline changelogs/fragments/6919.yml:2]:
```suggestion
- [MDS] Allow adding sample data for Timeline visualizations ([#6919](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6919))
```

**Comment by AMoo-Miki** [comment]
Thanks @zhongnansu  and @zhyuanqi

**Comment by zhongnansu** [comment]
> Thanks @zhongnansu and @zhyuanqi

Thanks for the suggestions!

**Review by zhyuanqi — APPROVED** [review]

**Review by BionIT — APPROVED** [review]

**Pull request merged**